arch/arm/boot/dts/overlays/README

```diff
@@ -1346,6 +1346,17 @@ Params: card_name               Override the default, "IQAudIODigi", card name.
         dai_stream_name         Override the default, "IQAudIO Digi HiFi",
                                 dai stream name.
 
```

Comment:
Sorry, but there must be two blank lines between entries.

Reply:
ok, done.